src/models/helpers/escapeFields.js

```diff
@@ -40,10 +44,13 @@ export function escapeDataFields(instance, fields) {
 export default function escapeFields(instance, fields) {
   const changed = instance.changed();
```

Comment:
just to circle back to this, before we go any further with this: I think, based on the errors I saw a couple of weeks ago when I was debugging this in the CI, that we need to make sure changed exists/is a function/however we want to bubble wrap it